rc/detection/editorconfig.kak

```diff
@@ -20,13 +20,13 @@ define-command editorconfig-load -params ..1 -docstring "editorconfig-load [file
             /*) # $kak_buffile is a full path that starts with a '/'
                 printf %s\\n "remove-hooks buffer editorconfig-hooks"
                 editorconfig "$file" | awk -v file="$file" -F= -- '
-                    /indent_style=/             { indent_style = $2 }
-                    /indent_size=/              { indent_size = $2 == "tab" ? 4 : $2  }
```

Comment:
Do we not need the ternary comparison any more?

Reply:
It's duplicated below

Reply:
From what I understand, if `indent_size="tab"` and `tab_width` is not set, then line 41 in the `END` scope will print out `set-option buffer tabstop tab`.

Reply:
Good point, I'll remove the other one instead